pkg/model/restore_result.go

```diff
@@ -0,0 +1,34 @@
+package model
+
+type Status string
```

Comment:
It's better to rename it to `JobStatus` to be more specific, since it is located in the model package.